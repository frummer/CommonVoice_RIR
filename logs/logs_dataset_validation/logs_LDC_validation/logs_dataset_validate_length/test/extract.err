Processing subdirectories:   0%|          | 0/6 [00:00<?, ?dir/s]Processing subdirectories:  17%|█▋        | 1/6 [00:00<00:02,  2.45dir/s]Processing subdirectories:  33%|███▎      | 2/6 [00:00<00:01,  3.45dir/s]Processing subdirectories:  50%|█████     | 3/6 [00:00<00:00,  3.36dir/s]Processing subdirectories:  67%|██████▋   | 4/6 [00:01<00:00,  3.00dir/s]Processing subdirectories:  83%|████████▎ | 5/6 [00:01<00:00,  3.26dir/s]Processing subdirectories: 100%|██████████| 6/6 [00:01<00:00,  3.73dir/s]
