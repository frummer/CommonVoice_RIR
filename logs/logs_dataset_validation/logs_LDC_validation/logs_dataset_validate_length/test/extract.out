Validating Datasets subdirs length
Script started at: Mon Feb 10 07:48:58 PM EST 2025
Base Directory: /export/fs05/afrumme1/babylon_datasets_1/LDC2014S02/V1_split/test
mixture: 677 .wav files
compressed_mixture: 677 .wav files
source1_reverb: 677 .wav files
source2_reverb: 677 .wav files
source2: 677 .wav files
source1: 677 .wav files
